apps/codeforafrica/src/lib/data/blockify/ourTeam.js

```diff
@@ -1,23 +1,34 @@
+import { countries } from "@/codeforafrica/lib/data/json/countries";
 import { getMembers } from "@/codeforafrica/lib/data/utils/members";
+import { sortTags } from "@/codeforafrica/lib/data/utils/tags";
+import equalsIgnoreCase from "@/codeforafrica/utils/equalsIgnoreCase";
+
+const getCountryFromName = (name) =>
+  countries.find((c) => equalsIgnoreCase(c.slug, name)) ?? null;
+
+function getTeamTags(docs) {
+  const tags = sortTags(docs.map((item) => item.team).filter(Boolean));
+  return [{ name: "All", slug: "All" }, ...tags];
```

Comment:
You can declare `const ALL_TAG = { name: "All", slug: "All" };` somewhere and reuse it... may be in `utils/tags` or something.